x-pack/plugins/synthetics/common/rules/synthetics/translations.ts

```diff
@@ -11,48 +11,51 @@ export const SyntheticsMonitorStatusTranslations = {
   defaultActionMessage: i18n.translate(
     'xpack.synthetics.alerts.syntheticsMonitorStatus.defaultActionMessage',
     {
-      defaultMessage:
-        'The monitor {monitorName} checking {monitorUrl} from {locationName} last ran at {checkedAt} and is {status}. The last error received is: {lastErrorMessage}.',
+      defaultMessage: `{monitorName} {monitorType} monitor is {status} from {locationName} at {checkedAt}.\nThe error received is: "{lastErrorMessage}".\n{linkMessage}`,
       values: {
         monitorName: '{{context.monitorName}}',
-        monitorUrl: '{{{context.monitorUrl}}}',
+        monitorType: '{{context.monitorType}}',
         status: '{{{context.status}}}',
         lastErrorMessage: '{{{context.lastErrorMessage}}}',
         locationName: '{{context.locationName}}',
         checkedAt: '{{context.checkedAt}}',
+        linkMessage: '{{{context.linkMessage}}}',
       },
     }
   ),
   defaultSubjectMessage: i18n.translate(
     'xpack.synthetics.alerts.syntheticsMonitorStatus.defaultSubjectMessage',
     {
-      defaultMessage: 'The monitor {monitorName} checking {monitorUrl} is down.',
+      defaultMessage: '{monitorName} ({locationName}) is Down - Elastic Synthetics',
       values: {
         monitorName: '{{context.monitorName}}',
-        monitorUrl: '{{{context.monitorUrl}}}',
+        locationName: '{{context.locationName}}',
       },
     }
   ),
   defaultRecoverySubjectMessage: i18n.translate(
     'xpack.synthetics.alerts.syntheticsMonitorStatus.defaultRecoverySubjectMessage',
     {
-      defaultMessage: 'The monitor {monitorName} checking {monitorUrl} has recovered.',
+      defaultMessage: '{monitorName} ({locationName}) {recoveryStatus} - Elastic Synthetics',
       values: {
+        recoveryStatus: '{{context.recoveryStatus}}',
+        locationName: '{{context.locationName}}',
         monitorName: '{{context.monitorName}}',
-        monitorUrl: '{{{context.monitorUrl}}}',
       },
     }
   ),
   defaultRecoveryMessage: i18n.translate(
     'xpack.synthetics.alerts.syntheticsMonitorStatus.defaultRecoveryMessage',
     {
       defaultMessage:
-        'The alert for the monitor {monitorName} checking {monitorUrl} from {locationName} is no longer active: {recoveryReason}.',
+        'The alert for {monitorName} ({locationName}) {monitorType} monitor is no longer active: {recoveryReason}.\nThe latest error received was: "{lastErrorMessage}".\n{linkMessage}',
```

Comment:
@florent-leborgne not specified in the AC, but this was the best generic content I could come up with for an alert recovery, given that there are 3 recovery paths 1. monitor was deleted 2. monitor location was removed or 3. monitor received an up status. The initial content is generic while the `recoveryReason` contains the specifics.

Reply:
LGTM, similar to my other comment, maybe have some light formatting between the name, location, and type like hyphens or commas, just so it's easier to read and differentiates the characteristics